src/msg/Message.h

```diff
@@ -80,6 +80,7 @@
 #define MSG_OSD_MARK_ME_DOWN 74
 #define MSG_OSD_FULL         75
 #define MSG_OSD_MARK_ME_DEAD 123
+#define MSG_OSD_MARK_ME_DOWN_AND_DEAD 124
```

Comment:
Does this require a new message type? It seems like we could make processing in the mon conditional on the down_and_dead field instead of adding a new type.

Reply:
you are right, we can also get rid of a few other things if we will re-use the old message type and use the down_and_dead
I made the changes